docs/integrations/sources/surveymonkey.md

```diff
@@ -58,35 +74,36 @@ Please read this [docs](https://developer.surveymonkey.com/api/v3/#getting-start
 
 The SurveyMonkey API applies heavy API quotas for default private apps, which have the following limits:
 
-* 125 requests per minute
-* 500 requests per day
+* 125 requests per minute.
+* 500 requests per day.
 
-To cover more data from this source we use caching.
+Use caching to cover more data from this source.
 
 ## Changelog
 
 | Version | Date       | Pull Request                                             | Subject                                                                          |
 |:--------|:-----------|:---------------------------------------------------------|:---------------------------------------------------------------------------------|
+| 0.3.1   | 2024-05-04 | [123](https://github.com/airbytehq/airbyte/pull/xyz)   | Apply editorial review
```

Comment:
1. Fixed spelling and grammar errors.

2. Added a new row for this editorial review PR.